### PR TITLE
Update config-settings.rst

### PR DESCRIPTION
**Pull request by wiersgallak**
https://github.com/mattermost/mattermost-webapp/pull/3033 & addition of team & channel pages

### CODE REVIEW

File: source/administration/config-settings.rst [hunk 1]
```diff
@@ -75,9 +75,21 @@ Settings for managing users, user access, and permissions.
 
 Users
 ~~~~~~~~~~~~~~~~~~~~~~~~~
-View and manage active and inactive users. 
+View and manage active and inactive users. Also provides the ability the admin to revoke all user sessions.
```
Comment (jasonblais) [inline source/administration/config-settings.rst]:
```suggestion
View and manage active and inactive users, and revoke all user sessions. 
```

File: source/administration/config-settings.rst [hunk 2]
```diff
@@ -2577,6 +2589,19 @@ It is recommended to use ``"https://login.microsoftonline.com/common/oauth2/v2.0
 | This feature's ``config.json`` setting is ``"TokenEndpoint": "https://login.microsoftonline.com/common/oauth2/v2.0/token"`` with string input.                       |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Guest Access (Beta)
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Enable Guest Access
+^^^^^^^^^^^^^^^^^^^
+
+**True**: Allow guest invitations to channels within teams.  Please see Guest documenation for more information.
```
Comment (amyblais) [inline source/administration/config-settings.rst]:
```suggestion
**True**: Allow guest invitations to channels within teams. Please see Guest documentation for more information. 
```

**Comment by wiersgallak** [comment]
@amyblais @jasonblais Sorry about the timing of adding reviewers earlier.  I had realized afterwards that I had not gotten the Guest Account config settings added.  Should be good to go now.

File: source/administration/config-settings.rst [hunk 3]
```diff
@@ -2577,6 +2589,41 @@ It is recommended to use ``"https://login.microsoftonline.com/common/oauth2/v2.0
 | This feature's ``config.json`` setting is ``"TokenEndpoint": "https://login.microsoftonline.com/common/oauth2/v2.0/token"`` with string input.                       |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Guest Access (Beta)
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Enable Guest Access
+^^^^^^^^^^^^^^^^^^^
+
+**True**: Allow guest invitations to channels within teams. Please see `Guest Accounts documenation <>`_ for more information.
```
Comment (amyblais) [inline source/administration/config-settings.rst]:
```suggestion
**True**: Allow guest invitations to channels within teams. Please see :doc:`Guest Accounts documentation <guest-accounts>`_ for more information.
```

**Pull request merged**